src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -83,13 +86,13 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
                 is RsTraitItem -> return
             }
         }
-        val (elem, desc) = when (field) {
-            is RsFieldLookup -> field to "Attempted to access a private field on a struct. [E0616]"
-            is RsMethodCall -> field.identifier to "A private item was used outside of its scope. [E0624]"
-            is RsPath -> field to "A private item was used outside its scope. [E0603]"
+        val error = when (field) {
```

Comment:
Maybe also rename `field` to simple `element`. It's not the only field now, but name remains old